examples/connectivity/compare_resting_state.py

```diff
@@ -0,0 +1,215 @@
+import glob
```

Comment:
Examples should always start with a module docstring that explains what is done.